presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -247,6 +247,7 @@ public ConnectorTableLayout getTableLayout(ConnectorSession session, ConnectorTa
                 Optional.empty(),
                 Optional.empty(),
                 Optional.empty(),
-                ImmutableList.of());
+                ImmutableList.of()
+        );
```

Comment:
undo this and below formatting.

As far as I know the convention is to keep `;` in the last line of the multi-line statements